CHANGES_NEXT_RELEASE

```diff
@@ -30,6 +30,7 @@
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
 - Fix: Distinguish between strings and numbers in q string filter (Issue #1129)
 - Fix: proper response for 'GET /v2/entities/<id>?attrs=<attrName>' and 'GET /v2/entities/<id>/attrs?attrs=<attrName>' when <attrName> doesn't exist (#2241)
+- Fix: throttling set to zero (Issue #2030)
```

Comment:
Yes, this is another one of those :-(
Will perform those tests and hopefully I can tell when the bug was fixed ...